test/integration/models/test_vpc.py

```diff
@@ -42,3 +41,57 @@ def test_update_subnet(get_client, create_vpc_with_subnet):
     subnet = get_client.load(VPCSubnet, subnet.id, vpc.id)
 
     assert subnet.label == new_label
+
+
+def test_fails_create_vpc_invalid_data(get_client):
+    try:
+        get_client.vpcs.create(
+            label="invalid_label!!",
+            region=get_region(get_client, {"VPCs"}),
+            description="test description",
+        )
+    except ApiError as e:
+        assert e.status == 400
+        assert "Label must include only ASCII" in str(e.json)
```

Comment:
```suggestion
    with pytest.raises(ApiError) as excinfo:
        get_client.vpcs.create(
            label="invalid_label!!",
            region=get_region(get_client, {"VPCs"}),
            description="test description",
        )
    assert e.status == 400
    assert "Label must include only ASCII" in str(e.json)
```

Can we use the `pytest.raises` to ensure the exception is raised?

Reply:
Looks cleaner this way. I will make the change